frontend/public/scss/product-details.scss

```diff
@@ -234,6 +234,21 @@
         }
       }
 
+      .more-dates {
+        padding-bottom: 6px;
+        width: 100%;
+        font-size: 12px;
+
+        .dates-tooltip {
+          color: $link-blue !important;
+          text-decoration: underline;
+        }
+      }
+
```

Comment:
The tooltip title shows with a grey background.  I wonder if it's possible to remove the background using something like what's shared here: https://stackoverflow.com/questions/25390463/how-to-change-the-color-of-background-of-title-tag